blocks/link-editor/index.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import { isBoolean, noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { IconButton } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import UrlInput from '../url-input';
+import { filterURLForDisplay } from '../../editor/utils/url';
+import ToggleControl from '../inspector-controls/toggle-control';
+
+export default class LinkEditor extends Component {
```

Comment:
This PR is intended to unify the link control behavior, do you think we should reuse this component elsewhere (image, cover-image blocks, format toolbar?)

Reply:
Yes it makes sense to use it elsewhere my next step would be using it in format toolbar.